### PR TITLE
[WIP] doc(sdk): Add README to SDK

### PR DESCRIPTION
**Pull request by jhermsmeier**
This adds a README to the SDK's subdirectory, describing its
architecture and modules used, for a braod but comprehensive
overview.

Change-Type: patch

### CODE REVIEW

File: lib/sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Etcher SDK
+
+## Modules
+
+This section shall give a broad overview of the modules used by the Etcher SDK, and what they're used for.
```
Comment (lurch) [inline lib/sdk/README.md]: I guess this is currently listing just the _external_ modules in the Etcher SDK, and not (yet) including any of the modules in the Etcher repo itself?

**Comment by lurch** [comment]
`braod` typo in the commit-message ;)

**Comment by lurch** [comment]
CI builds are failing because `lib/sdk/README.md contains non-ASCII characters`

**Comment by jhermsmeier** [comment]
Oh my – the culprit was a dash;

**Comment by lurch** [comment]
Still the same commit-message typo though :laughing:

**Comment by jhermsmeier** [comment]
Oh no, I do that when writing, because `-` is a bit too short when you want a `–`; we should totally allow the full UTF range for markdown files I think. But that's for another time & place.
Regarding the commit msg; yeah, I'll fix that when I finish the doc :)

**Comment by lurch** [comment]
> we should totally allow the full UTF range for markdown files I think. But that's for another time & place.

Just did a quick search, and https://stackoverflow.com/questions/115210/how-to-check-whether-a-file-is-valid-utf-8 has some options that'd be useful for that.

**Comment by jhermsmeier** [comment]
Closing, as to be obsoleted by SDK reworking

**Pull request closed without merging**